arch/xtensa/src/esp32/hardware/esp32_soc.h

```diff
@@ -183,6 +183,14 @@
 
 #define GET_PERI_REG_BITS2(reg, mask,shift)      ((READ_PERI_REG(reg)>>(shift))&(mask))
 
+/* Extract the field from the register and shift it to avoid wrong reading */
+
+#define REG_MASK(_reg, _field) ((_reg & (_field##_M)) >> (_field##_S))
```

Comment:
```suggestion
#define REG_MASK(_reg, _field) (((_reg) & (_field##_M)) >> (_field##_S))
```
Wrap macro argument in parentheses to avoid surprises with operator precedence

Reply:
done